messaging/src/main/java/org/axonframework/eventhandling/pooled/WorkPackage.java

```diff
@@ -89,6 +89,8 @@ class WorkPackage {
     private TrackingToken lastConsumedToken;
     private TrackingToken lastStoredToken;
     private long lastClaimExtension;
+    private final String segmentIdResourceKey;
```

Comment:
Super nit: if you move these fields next to the other `private final` fields (so below the `Clock`), you'd make me extremely happy. :angel: